src/Compilers/Core/Portable/Symbols/INamespaceSymbol.cs

```diff
@@ -27,6 +28,11 @@ public interface INamespaceSymbol : INamespaceOrTypeSymbol
         /// </summary>
         new IEnumerable<INamespaceOrTypeSymbol> GetMembers(string name);
 
+        /// <summary>
+        /// Get all the members of this symbol that have a particular name.
+        /// </summary>
+        void GetMembers<TArg>(string name, Action<INamespaceOrTypeSymbol, TArg> callback, TArg argument);
```

Comment:
i don't have an issue with this.  but my guess is that we need to be consistent with this and the other cases of Getmembers we have.  want to extract out and bring to API review?